### PR TITLE
Add workflow for Shibarium

### PR DESCRIPTION
**Pull request by varasev**

### CODE REVIEW

**Review by sl1depengwyn — COMMENTED** [review]
Shouldn't we add changelog entry under the `chore` section?

File: .github/workflows/publish-docker-image-for-shibarium.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./docker/Dockerfile
+          push: true
+          tags: blockscout/blockscout-${{ env.DOCKER_CHAIN_NAME }}:${{ env.RELEASE_VERSION }}-postrelease-${{ env.SHORT_SHA }}
+          platforms: |
+            linux/amd64
+            linux/arm64/v8
```
Comment (nikitosing) [inline .github/workflows/publish-docker-image-for-shibarium.yaml:54]: Do we really need to build it for arm?

**Comment by nikitosing** [comment]
@varasev We should add `build and push docker image` to `.github/workflows/release-additional.yml`
Also, new workflow, won't work since branch `production-shibarium` does not exist

File: .github/workflows/publish-docker-image-for-shibarium.yaml (hunk 1 above)
Comment (nzenchik) [reply .github/workflows/publish-docker-image-for-shibarium.yaml]: why not? we may run ourselves arm hosts or someone else can run this image on arm

Comment (nikitosing) [reply .github/workflows/publish-docker-image-for-shibarium.yaml]: usually we build arm images only for releases

**Comment by varasev** [comment]
> Shouldn't we add changelog entry under the `chore` section?

Done: https://github.com/blockscout/blockscout/pull/9303/commits/4b777b7a1099e1cef11a5ae74d1a18dc15c91af7

**Review by nikitosing — APPROVED** [review]

**Review by sl1depengwyn — APPROVED** [review]

File: .github/workflows/publish-docker-image-for-shibarium.yaml (hunk 1 above)
Comment (varasev) [reply .github/workflows/publish-docker-image-for-shibarium.yaml]: I removed arm for the prerelease: https://github.com/blockscout/blockscout/pull/9303/commits/1a9a2886825a924775e818ee8cba037fcba616db

**Pull request merged**